css-two-column-layout/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-1">
+        <img src="./images/software.svg" alt="software" title="software">
+      </div>
+      <div class="column-info">
+        <h3>INTUITIVE, SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilitites run across Africa - making them radically more efficient operationaly and financially, a complete turnaround.</p>
+      </div>
+      <div class="column-1">
+        <img src="./images/practice.svg" alt="practice" title="practice">
+      </div>
+      <div class="column-info">
+        <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+        <p>Our all-in-one health records (EHR), hospital management (HMIS) &amp; billing system helps streamline your facility, reduce waste and improve quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-2">
+        <img src="./images/circumstance.svg" alt="atom" title="atom">
+      </div>
+      <div class="column-info">
+        <h3>FOR ANY CIRCUMSTANCE</h3>
+        <p>Our intuitive system is built around the peculiarities of your facility; we support offline deployments, sync to the cloud and run on any device 24/7.</p>
+      </div>
+      <div class="column-2">
+        <img src="./images/data.svg" alt="data" title="data">
+      </div>
+      <div class="column-info">
+        <h3>DATA THAT TRANSFORMS</h3>
+        <p>We understand your facility better than you do; we provide visually-appealing real-time analytics, reports and trends at your finger tips.</p>
+      </div>
+    </div>
+    <div>
+      <div class="button">
+        <button class="button-learn-more">LEARN MORE</button>
+      </div>
+    </div>
+  </div>
+</body>
+</html>
```

Comment:
Can we rework this so that it follows the grid system with container, rows and columns?

Reply:
@KatherineBustamanteC ☝🏼 this as well